sdk/nodejs/output.ts

```diff
@@ -567,9 +639,9 @@ export interface OutputInstance<T> {
      * available for functions that end up executing in the cloud during runtime.  To get the value
      * of the Output during cloud runtime execution, use `get()`.
      */
-    apply<U>(func: (t: T) => Promise<U>): Output<U>;
-    apply<U>(func: (t: T) => OutputInstance<U>): Output<U>;
-    apply<U>(func: (t: T) => U): Output<U>;
+    apply<U>(func: (t: T) => Promise<U>, runWithUnknowns?: boolean): Output<U>;
+    apply<U>(func: (t: T) => OutputInstance<U>, runWithUnknowns?: boolean): Output<U>;
+    apply<U>(func: (t: T) => U, runWithUnknowns?: boolean): Output<U>;
```

Comment:
1. for now, let's remove this from the public API (unless we had an important downstream use case).
2. if we add this, let's doc.
3 i would prefer these as overloads.  i really don't want people to see these in the first three overloads if they don't need it.

i.e. htese are such a niche/specialized case, we should avoid advertising things as much as we can.

Reply:
If these are not listed here, we'll need some `<any>` casts peppered throughout this file. Is there a way to add overloads and keep them from being user-visible?

Reply:
>Is there a way to add overloads and keep them from being user-visible?

Not that i know of.  Another approach is to have an internal non-exported interface that has these overloads.  Then just cast to that.  Since this is all internal code i have no problem with that.

basically, it's a way for us to say "we know it's typesafe to peek in and do that, but we don't really want external consumers to do the same right now."

Reply:
Fixed.